docs/blog/2019-04-03-how-we-think-about-product-at-gatsby/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "How we think about product at Gatsby"
+date: 2019-04-03
```

Comment:
Should this date get updated to something more current? Or future dated for the blog schedule? I'm not quite sure how that works yet.

Reply:
It should get changed to the date that it's posted I think. I updated the date to today. :)